config/manifests/bases/network-observability-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,59 @@
+    mediatype: image/png
+  install:
+    spec:
+      deployments: null
+    strategy: ""
+  installModes:
+  - supported: false
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - network observability
+  - ipfix
+  - flow tracing
+  - flows
+  - network
+  - observability
+  links:
+  - name: Network Observability Operator
+    url: https://network-observability-operator.domain
+  maintainers:
+  - email: jtakvori@redhat.com
+    name: Joel Takvorian
+  - email: jpinsonn@redhat.com
+    name: Julien Pinsonneau
+  - email: mmaciasl@redhat.com
+    name: Mario Macias
+  - email: ocazade@redhat.com
+    name: Olivier Cazade
+  maturity: alpha
+  provider:
+    name: Red Hat
```

Comment:
Should we use "Red Hat" as provider or just "Network Observability" for upstream?